examples/basic/src/Basic.purs

```diff
@@ -119,3 +125,15 @@ moveItem fromIndex toIndex items =
     take toIndex items'
       <> maybe [] pure item
       <> drop toIndex items'
+
+main :: Effect Unit
+main = do
+  maybeContainer <- window
+    >>= document
+    >>= toNonElementParentNode
+      >>> getElementById "container"
+  case maybeContainer of
+    Nothing -> throw "Container element not found."
+    Just container -> do
+      todoExample <- mkTodoExample
+      render (todoExample unit) container
```

Comment:
This used to be in a separate `index.js`, and there was a Browserify command in the Makefile that would bundle it with the `purs` output (after writing `module.exports = PS.Basic;`). It seemed simpler to just use `spago bundle-app` with a single PS module.